main.pm

```diff
@@ -1,18 +1,5 @@
-# Copyright (C) 2014-2018 SUSE LLC
-#
-# This program is free software; you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation; either version 2 of the License, or
-# (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License along
-# with this program; if not, write to the Free Software Foundation, Inc.,
-# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+# Copyright 2014-2018 SUSE LLC
```

Comment:
Why not 2014-2021 since those files are being touched?

Reply:
hi. This is also discussed in https://progress.opensuse.org/issues/98616 . At least oorlov and me want to avoid the noisy changes introduced by "needing to bump the copyright year". In general only on "significant changes" one should update the year anyway and it's debatable when significant changes are introduced.

Commonly the copyright extends only if significant changes are applied to files
and even in that case renewal of the copyright is automatic based on the actual
content differences, not needing an explicit statement in the copyright header.
We usually extended the copyright with every new year as soon as changes have 
been made. This has been seen as a nuisance by many and causes unnecessary
changes in the git log.

Other references like https://opensource.google/docs/copyright/ and
https://matija.suklje.name/how-and-why-to-properly-write-copyright-statements-in-your-code
and https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/ also 
suggest to only include the original year of publication in the copyright
notice. In practice it would likely not matter if a 70 year period of copyright
is extended by just a handful of years anyway.

This was confirmed by our legal counseling.